pkg/driver/clickhouse/clickhouse.go

```diff
@@ -47,6 +49,33 @@ func connectionString(initialURL *url.URL) string {
 	u.Host = host
 
 	query := u.Query()
+
+	certPath := query.Get("cert_path")
```

Comment:
Is this a standard parameter used by other ClickHouse clients? I want to avoid dbmate-specific configuration where possible.